osdconfig/proto/config.proto

```diff
@@ -0,0 +1,120 @@
+syntax = "proto3";
+
+// this package defines the configuration parameters and interfaces
+// requires to run portwork services
+package proto;
+
+service Spec {
+    rpc GetGlobalSpec(Empty) returns (GlobalConfig) {}
+    rpc SetGlobalSpec(GlobalConfig) returns (Ack) {}
+    rpc GetClusterSpec(Empty) returns (ClusterConfig) {};
+    rpc SetClusterSpec(ClusterConfig) returns (Ack) {};
+    rpc GetNodeSpec(NodeID) returns (NodeConfig) {};
+    rpc SetNodeSpec(NodeConfig) returns (Ack) {};
+}
+
+message Empty {}
+message Ack {
+    int64 n = 1;
+}
+message NodeID {
```

Comment:
nit: newlines between these messages and for Empty we can write it - 
message Empty {
}

Reply:
sure... now only if they had made gofmt for proto files as well we would be saving ourselves some review time :)